src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/DataElementsCard.tsx

```diff
@@ -218,7 +218,7 @@ const DataElementsCard = (props: {
     setAttributesPresent(false);
     if (selectedDataElement && selectedDataElement.schema?.eachPath) {
       selectedDataElement.schema.eachPath((path) => {
-        if (!SKIP_ATTRIBUTES.includes(path) && !attributesPresent) {
```

Comment:
I think this was added for a different ticket..[https://jira.cms.gov/browse/MAT-5992](https://jira.cms.gov/browse/MAT-5992)

It sounds like the change broke attributes tab...but can you make sure 5992 is still met with the removal of this check? Or would we end up needing to reopen or re-create 5992?

Reply:
So i checked on a Payer element like MAT-5992 mentions and it looks to be working as expected still. 

The code to turn off the attributes is still there, this change just removes the extra check that really isn't necessary and seemed to be screwing up when adding codes.